go/adapter/otel_stdout/adapter.go

```diff
@@ -84,3 +87,11 @@ func (o *OtelStdoutAdapter) makeCallSpans(event observe.CallEvent, parentId *str
 
 	return spans
 }
+
+// We don't need any background task for this Adapter
+func (o *OtelStdoutAdapter) Start() {
+}
+
+// We don't need any background task for this Adapter
+func (o *OtelStdoutAdapter) Stop(wait bool) {
+}
```

Comment:
would remove these altogether vs. add comments . the comments will show up on public documentation, and the empty implementations shouldn't be needed since we embed the AdapterBase, they are already "on" the OtelStdoutAdapter